db/model/Variable.ts

```diff
@@ -141,65 +147,56 @@ export async function getVariableData(
             additionalInfo: partialSource.additionalInfo || "",
         },
     }
-    const variableData: OwidVariableMixedData = {
-        years: [],
-        entities: [],
-        values: [],
-    }
 
-    const entityMap = new Map<number, OwidVariableDimensionValueFull>()
-    const yearMap = new Map<number, OwidVariableDimensionValuePartial>()
+    const entities = await db.queryMysql(
+        `
+    SELECT
+        id,
+        name,
+        code
+    FROM entities WHERE id in (?)
```

Comment:
could potentially add an `ORDER BY id` here, but I don't know if `variableData.entities` is already sorted maybe?
Not totally necessary, but would make it easier whenever a human takes a look at the metadata file.

Reply:
Nice idea, added sorting by name. (By the way we're not sorting it at the moment at all)